reg_tests/test_files/airfoilRANSEdgeTrilinos/airfoilRANSEdgeTrilinos.i

```diff
@@ -0,0 +1,217 @@
+# -*- mode: yaml -*-
+#
+# 2-D RANS (k-omega SST) simulation of DU-91-W2-225 airfoil
+#
+# U = 15 m/s; aoa = 4 deg; Re = 2.0e6
+#
+
+Simulations:
+  - name: sim1
+    time_integrator: ti_1
+    optimizer: opt1
+
+linear_solvers:
+
+  - name: solve_scalar
+    type: tpetra
+    method: gmres
+    preconditioner: sgs
+    tolerance: 1e-5
+    max_iterations: 10
+    kspace: 75
+    output_level: 0
+    write_matrix_files: no
+
+  - name: solve_cont
+    type: tpetra
+    method: gmres
+    preconditioner: muelu
+    tolerance: 1e-5
+    max_iterations: 200
+    kspace: 75
+    output_level: 0
+    muelu_xml_file_name: ../../xml/milestone_aspect_ratio_gs.xml
+
+  - name: solve_mom
+    type: tpetra
+    method: gmres
+    preconditioner: sgs
+    tolerance: 1e-5
+    max_iterations: 200
+    kspace: 75
+    output_level: 0
+    segregated_solver: yes
+
+realms:
+
+  - name: realm_1
+    mesh: ../../mesh/du91w2_airfoil.exo
+    automatic_decomposition_type: rcb
+    use_edges: yes
+
+    time_step_control:
+     target_courant: 1000.0
+     time_step_change_factor: 1.05
+
+    equation_systems:
+      name: theEqSys
+      max_iterations: 2
+
+      solver_system_specification:
+        velocity: solve_scalar
```

Comment:
Question: Did you mean to use the segregated solver `solve_mom` (lines 35-43) here?

Reply:
Yes good, point, I will re-run these swapping the `solve_scalar` with `solve_mom` for the velocity field.
Thanks for noticing!